Reference/Debugging/index.md

```diff
@@ -0,0 +1,71 @@
+---
+versionFrom: 8.1.0
+---
+
+# Debugging
+In version Umbraco 8.1+ we have enabled a debugging feature from Microsoft & Visual Studio called SourceLink which allows you to step into the native Umbraco CMS source code.
+
+## Enabling SourceLink in Visual Studio 2019
```

Comment:
Is this the same in vs2017?

Reply:
I need to check & verify @marcemarc 90% sure it is

Reply:
```suggestion
## Enabling SourceLink in Visual Studio 2017 & 2019
```